CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 * Replace `explicit_bzero(3)` with `OPENSSL_cleanse(3)` to fix compilation on non-glibc systems. (see #75)
 * Replaced manual calculation of hash digest length by library function in ECIES
 * Improved error message when attempting to create an AES-GCM cipher with empty IV. (see #83)
+* Move DigestTypes from openssl namespace to mococrw namespace. This is an ABI change!
```

Comment:
Please remove the internal issue reference from the commit message. Also, we usually use present tense imperative, i.e. "Move", not "Moving".

Reply:
I don't get want you mean here.
Can you please be more verbose